TaskLayer/SearchTask/PostSearchAnalysisTask.cs

```diff
@@ -905,6 +909,149 @@ private void WritePeptideResults()
             }
         }
 
+        private void WriteVariantResults()
+        {
+            Status("Writing variant peptide results...", Parameters.SearchTaskId);
+            string variantPsmFile = Path.Combine(Parameters.OutputFolder, "VariantPSMs.psmtsv");
```

Comment:
Good catch! I'm interested in using these lists to calculate a group FDR at 1 % global FDR, but I really don't care about the PSM FDR within these files.

What do we think about recalculating the PSM FDR within this context? i.e. (decoy variants / target variants)

Reply:
I think there might be an issue though because, if I understand what Rob is saying, that the PSMs that we are calling 1% FDR are not actually 1% global FDR. It might be close, but there could be some differences. I think that we could remedy this by simply moving the WriteVariantResults() above WritePeptideResults(). I am all for calculating a group FDR in these files, but I think making sure that the psms we use are the actual 1 % global FDR PSMs from the allPSMs file is important.

Reply:
Oh, sounds good about writing them above `WritePeptideResults()`! 😄